src/Solnet.Rpc/Models/Logs.cs

```diff
@@ -34,4 +36,38 @@ public class LogInfo : Log
         /// </summary>
         public string Signature { get; set; }
     }
+
+    public class InstructionError
+    {
+        public int ErrorCode { get; set; }
+        
+        public KeyValuePair<string, int> CustomError { get; set; }
+    }
+    
+    public class InstructionErrors
+    {
+        [JsonPropertyName("InstructionError")]
```

Comment:
Redundant. Doesn't diverge from the property name

Reply:
Not because of the serializer options